frontend/packages/kubevirt-plugin/src/components/create-vm/forms/boot-source-form.tsx

```diff
@@ -180,9 +183,12 @@ const AdvancedSection: React.FC<AdvancedSectionProps> = ({
                   });
                 }}
                 provisioner={provisioner}
-                accessMode={state.accessMode?.value}
+                accessMode={state.accessMode?.value || spAccessMode?.getValue()} // Getting exitsting accessMode value otherwise
```

Comment:
nitpick: exitsiting => existing

Sorry, I missed this one in the last round.

Reply:
no you didn't missed it, it's been added in last iteration but nice catch lol, I'll get it fixed